### PR TITLE
[SCRT-27/28] Add more details on the viewing key creation process & UI enhancements

### PR DESCRIPTION
**Pull request by yoon-bbox**
Changes made:

1. Added a brief instruction on what information they should provide the provider with (viewing key + wallet address) - Tooltip
Screen Shot 2022-04-14 at 2 41 01 PM

2. Added an ability for the user to download the generated viewing key (like we do with permit)

3. Made the modal a bit more consistent with the query permit modal, applied custom fonts

Screen Shot 2022-04-14 at 2 41 30 PM
Screen Shot 2022-04-14 at 2 59 36 PM

4. Fixed the z - index of the background image (diagonal waves)
5. Added a hover effect on the link button
6. Disabled the header shrink animation on the main flow

**Note:** Haven't figured out what's causing this weird scroll bar issue.. I tried to add overflow-x-hideen/overflow-hidden in the layout component but didn't fix it so I left it for now..

https://user-images.githubusercontent.com/82831286/163482139-4ab48b42-00d4-4876-bcfa-8b4f6538f400.mov

### CODE REVIEW

File: src/components/header/index.tsx [hunk 1]
```diff
@@ -19,11 +19,11 @@ export default function Header() {
 
   const handleScroll = useCallback(() => {
     setTimeout(() => {
-      if (window.scrollY < 1) {
+      if (window.scrollY < 1 || !routerIsMain) {
         setShrinkHeader(false);
       } else setShrinkHeader(true);
     }, 200);
-  }, []);
+  }, [router]);
```
Comment (mmortelliti) [inline src/components/header/index.tsx:26]: should routerIsMain be here? What are you attempting with this component?

**Review by mmortelliti — CHANGES_REQUESTED** [review]
one question, but can't approve until tested when the testnet is back up

File: src/components/header/index.tsx (hunk 1 above)
Comment (yoon-bbox) [reply src/components/header/index.tsx:26]: I wanted to disable the shrinkHeader animation when it's not on '/' or  '/learn'

Comment (yoon-bbox) [reply src/components/header/index.tsx:26]: I wanted to disable the shrinkHeader animation when it's not on '/' or '/learn'

File: src/pages/applicant/viewingKey.tsx [hunk 2]
```diff
@@ -115,9 +152,21 @@ const ViewingKeyPage = () => {
     <div className="px-10 sm:px-14 z-50 mt-20 mb-20 sm:mt-20 md:-mt-8">
       <div className="w-full text-center">
         <div className="z-50 opacity-100 px-0 sm:p-10 flex flex-col">
-          <h2 className="z-50 font-semibold text-2xl sm:text-3xl md:text-3xl lg:text-4xl p-0">
-            {'Create a Viewing Key'}
-          </h2>
+          <div>
```
Comment (mmortelliti) [inline src/pages/applicant/viewingKey.tsx:155]: the icon is not clickable and the text is now behind the white lines. You may have to add zindex and/or change the display

**Review by mmortelliti — CHANGES_REQUESTED** [review]
minor fix

**Review by mmortelliti — APPROVED** [review]
lgtm!

**Pull request merged**